index.html

```diff
@@ -0,0 +1,302 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>
+      Digital Identities
+    </title>
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class=
+    "remove"></script>
+    <script class="remove">
+    "use strict";
+    // See https://github.com/w3c/respec/wiki/ for how to configure ReSpec
+    var respecConfig = {
+      github: "WICG/digital-credentials",
+      editors: [
+        {
+          name: "Marcos Caceres",
+          email: "marcosc@apple.com",
+          company: "Apple Inc.",
+          companyURL: "https://apple.com",
+        },
+        {
+          name: "Sam Goto",
+          email: "goto@google.com",
+          company: "Google Inc.",
+          companyURL: "https://google.com",
+        },
+        // Add additional editors here.
+        // https://github.com/w3c/respec/wiki/editors
+      ],
+      shortName: "digital-identity",
+      specStatus: "CG-DRAFT",
+      group: "wicg",
+      localBiblio: {
+      },
+      xref: {
+        profile: "web-platform",
+      },
+    };
+    </script>
+  </head>
+  <body data-cite="credential-management-1 WebCryptoAPI">
+    <section id="abstract">
+      <p>
+        This document specifies an API to allow user agents to mediate access
+        to, and representing, a digital identity (e.g. a digital driver's
+        license). The API builds on [[[credential-management-1]]] as a means to
+        request a digital identity from the user agent or underlying platform.
+      </p>
+    </section>
+    <section id="sotd">
+      <p>
+        This is an unofficial proposal.
+      </p>
+    </section>
+    <h2>
+      Introduction
+    </h2>
+    <p>
+      TBW
+    </p>
+    <h2>
+      Model
+    </h2>
+    <dl class="definitions" data-sort="">
+      <dt>
+        <dfn>Identity credential</dfn>
+      </dt>
+      <dd>
+        <p>
+          A specialized type of [=credential=] corresponding to the real-world
+          identity of a person enabling developers to make authentication
+          decisions based on verified identity information.
```

Comment:
```suggestion
          decisions based on verified identity statements made by an [=Issuer=].
```

Reply:
This distinction is important. Verifiable Credentials do not provide any indication of the veracity of the statements they contain; they only offer verification that the Issuer made those statements. 
```suggestion
          decisions based on identity statements verifiably made by an [=Issuer=].
```

Reply:
Good point, @TallTed.